bin/start_be.sh

```diff
@@ -233,7 +233,8 @@ set_tcmalloc_heap_limit() {
 ## set hdfs conf
 export LIBHDFS3_CONF="${DORIS_HOME}/conf/hdfs-site.xml"
 
-export MALLOC_CONF="percpu_arena:percpu,background_thread:true,metadata_thp:auto,muzzy_decay_ms:30000,dirty_decay_ms:30000,oversize_threshold:0,lg_tcache_max:16"
+# see https://github.com/jemalloc/jemalloc/issues/2366
+export JEMALLOC_CONF="percpu_arena:percpu,background_thread:true,metadata_thp:auto,muzzy_decay_ms:30000,dirty_decay_ms:30000,oversize_threshold:0,lg_tcache_max:16,prof:true,prof_prefix:jeprof.out"
```

Comment:
`prof:true` opening the profile seems to have an impact on performance.

can explain how to use jemalloc heap profile in `debug-tool.md`, which is currently based on tcmalloc.

![image](https://user-images.githubusercontent.com/13197424/209611558-09a75210-52d0-47b6-8e1d-abd886bf4ac2.png)

Reply:
Only set `prof:true` will not generate dump file unless we call the `mallctl` function in the doris code. 
It seems that it will not affect performance. 
i think we can keep this item in conf, which needs to be paid attention to in subsequent performance tests.

In the nex pr, i will add a http interface to generate the heap dump file at runtime and enrich `debug-tool.md`.